docs/devguide/README.md

```diff
@@ -0,0 +1,84 @@
+
+Install it, start it and create a new workspace for your JMC work. Creating a new workspace is as easy as picking a new name when starting up your Eclipse in the dialog asking for a directory for the workspace:
+
+![Workspace Selection](images/workspace.png)
+
+## Installing JDKs
+If you haven't already, you should now first build JMC using the instructions in the [README.md](../../README.md). 
+
+Next set up your JDKs in your Eclipse. Download and install a JDK 8 and JDK 11 (or the very latest update of the latest JDK), then open Window | Preferences and then select Java / Installed JREs. Add your favourite JKD 8 and JDK 11 JDKs (Add…) and then use Java / Installed JREs / Execution Environments to set them as defaults for the JDK 8 and JDK 11 execution environments.
+
+Setting installed JREs:
+
+![Set Installed JRE](images/setinstalledjre.png)
+
+Setting execution environments:
+
+![Set Execution Environment](images/setexecutionenvironment.png)
+
+Okay, we now have our JDKs set up. If you want to import not only the JMC core project, you will next need to set up a user library for things that JMC will need from the JDK. Go to Preferences | Java/Build Path/User Libraries and create a new library named JMC_JDK. Add (Add External JARs…) the following JARs from a JDK 8 (u40 or above) to the User Library: tools.jar (/lib/tools.jar) and finally jconsole.jar (/lib/jconsole.jar).
+
+Creating the user library:
+
+![Create User Library](images/createuserlibrary.png)
+
+Adding the jars:
+
+![image](images/addingjars.png)
+
+Now we need to check a few things…
+
+### Checkpoint
+* Is the Jetty server from the build instructions still up and running? ![](images/p2site.png)
+
+
+If yes, go ahead and open up the most recent target file you can find, available under releng/platform-definitions/platform-definition-&lt;year&gt;-&lt;month&gt; (File | Open File). You should see something like this:
+
+![Platform Target](images/platformtarget.png)
+
+Wait for all the dependencies to be properly loaded (check the progress in the lower right corner), then click the Set as Active Target Platform link in the upper right corner.
+
+Now there is one final preparation for the import – we need to turn off certain Maven settings. Go to the preferences, and select Maven / Errors/Warnings. Set Plugin execution not covered by lifecycle configuration to Ignore, and also Out-of-date project configuration to Ignore
+
+![Maven Settings](images/mavensettings.png)
+
+Now the preparations are done, and we can start importing the projects. Woho!
+
+## Importing the Projects
+First we will have to import the core projects, since they are built separately from the rest. Select File | Import… and select Maven / Existing Maven Project.
+
+![Import Maven Project](images/importmaven.png)
+
+Click next, and browse into the jmc/core folder. Select all the core projects and import them.
+
+Next select File | Import… and select Maven / Existing Maven Project again, but this time from the root.
```

Comment:
I've been able to successfully follow the steps up to this point, but here I got an error:

<img width="410" alt="Screenshot 2020-06-10 at 23 04 34" src="https://user-images.githubusercontent.com/348973/84486996-0e1bdb80-ac96-11ea-8ae0-5ace8308fd7d.png">

After importing the Maven projects from the jmc root directory I see compilation errors because of a missing jface `ResourceLocator` path:

<img width="1171" alt="Screenshot 2020-06-11 at 00 31 22" src="https://user-images.githubusercontent.com/348973/84487139-39062f80-ac96-11ea-823b-f15f4f4b00a2.png">

The jar is there but it looks like an old version: `org.eclipse.jface_3.14.0.v20180423-0714.jar`. I've tried manually upgrading this jar and I've tried upgrading the plugin development environment but it didn't work - I don't know much about eclipse, I need to dig into this a bit more.

Reply:
I'm running:

```
Eclipse IDE for Eclipse Committers

Version: 2020-03 (4.15.0)
Build id: 20200313-1211
```